test-requirements.txt

```diff
@@ -13,7 +13,7 @@ requests
 maturin
 pytest-lazy-fixtures
 pytest-mock
-grpc-stubs==1.24.7
+grpc-stubs>=1.24.7
```

Comment:
Maybe match the version of the grpc packages here?

I mean, `>=1.24.7` happens to work because in new envs or if you force the upgrade because it defaults to installing the latest version, but it's technically not correct.

Reply:
Yeah, something like `>=1.44.0`